pkg/asset/installconfig/alibabacloud/client.go

```diff
@@ -262,6 +275,23 @@ func (client *Client) GetOSSObjectSignURL(bucketName string, objectName string)
 	return
 }
 
+// GetAvailabilityZonesByInstanceType returns a list of supported zones for the specified instance type.
```

Comment:
Please improve on this function description to make it clear how this function differs from `DescribeAvailableZoneByInstanceType`.